server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -2360,7 +2360,7 @@ public void run() {
         zoo.putPersistentData(zPath, new byte[0], NodeExistsPolicy.SKIP);
 
         if (tabletServerLock.tryLock(lw, lockContent)) {
-          log.debug("Obtained tablet server lock " + tabletServerLock.getLockPath());
+          log.info("Obtained tablet server lock " + tabletServerLock.getLockPath());
```

Comment:
I checked it out, I tried using the .getLockPath() method on all the services, but master and gc didn't return anything, so I took it out of those, and left it for the monitor and tservers.